### PR TITLE
👷 Add Publish Stage to Jenkinsfile

### PR DESCRIPTION
**Pull request by Paulomart**
## What did you change?

Now when Jenkins runs the Pipeline and the current branch is master,
it will publish the npm package to the registry.
The token for publishing is configured in Jenkins.

## How can others test the changes?

n/a

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

**Comment by Paulomart** [comment]
Currently the build will fail when the version is already published to npm. I could add `|| true` to ignore the result of `npm publish`

File: _templates/Jenkinsfile [hunk 1]
```diff
@@ -51,5 +51,16 @@ pipeline {
...
       }
     }
+    stage('publish') {
+      when {
+        branch 'master'
+      }
+      steps {
+        nodejs(configId: 'process-engine-ci-token', nodeJSInstallationName: 'node-lts') {
+          sh 'node --version'
+          sh 'npm publish --ignore-scripts'
```
Comment (cmg-dev) [inline _templates/Jenkinsfile]: So bitte.

```
// Kommentar
sh 'npm publish --ignore-scripts || true'`
```

Comment (heikomat) [reply _templates/Jenkinsfile]: @cmg-dev warum? mit git flow sollte der master immer gebaut werden können. Wenn nicht, sollte die info nicht verschluckt werden, oder?

Comment (Paulomart) [reply _templates/Jenkinsfile]: @cmg-dev Ich gebe @heikomat Recht, wir sollten diesen Fehler nicht verschlucken. Commit auf Master = Version Bump.

**Pull request merged**